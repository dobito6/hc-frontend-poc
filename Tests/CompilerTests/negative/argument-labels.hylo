//! no-std

struct A { memberwise init }

fun one(x: A) {}
fun one(x: A, y: A) {}

fun main() {
  one(y: A())
}
