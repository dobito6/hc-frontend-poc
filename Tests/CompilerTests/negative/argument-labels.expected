negative/argument-labels.hylo:7.3-6: error: no candidate matches the argument list
  one(y: A())
  ~~~
negative/argument-labels.hylo:3.5-8: note: incompatible labels: found '(x:)', expected '(y:)'
fun one(x: A) {}
    ~~~
negative/argument-labels.hylo:4.5-8: note: incompatible labels: found '(x:y:)', expected '(y:)'
fun one(x: A, y: A) {}
    ~~~
